tools/SetupFlow/DevHome.SetupFlow/ViewModels/MainPageViewModel.cs

```diff
@@ -82,6 +91,47 @@ public partial class MainPageViewModel : SetupPageViewModelBase
         ShowDevDriveItem = DevDriveUtil.IsDevDriveFeatureEnabled;
 
         BannerViewModel = bannerViewModel;
+
+        // If the feature is turned on, it doesn't show up in the configuration section (toggling it off and on again fixes it)
+        // It's because this is constructed after ExperimentalFeaturesViewModel, so the handler isn't added yet.
+        _host.GetService<IExperimentationService>().ExperimentalFeatures.FirstOrDefault(f => string.Equals(f.Id, QuickstartPlaygroundFlowFeatureName, StringComparison.Ordinal))!.PropertyChanged += ExperimentalFeaturesViewModel_PropertyChanged;

```

Comment:
For future update:

Line 42 has a private member variable called _experimentation Service. We can use the service directly with it. 

Also we can probably add a method to `IExperimentationService` to return the `ExperimentalFeature` object based on a name passed in. We'd move the linq statement into the method and then callers like here have a cleaner way to get the `IExperimentationService`

However, if we're augmenting the  `IExperimentationService` feature, at the same time we should add an event, callers can subscribe to get `ExperimentalFeature` changes